maps/submaps/surface_submaps/wilderness/wilderness.dm

```diff
@@ -192,4 +193,10 @@
 	name = "Musk"
 	desc = "0 to 60 in 1.9 seconds."
 	mappath = 'maps/submaps/surface_submaps/wilderness/Musk.dmm'
-	cost = 10
\ No newline at end of file
+	cost = 10
+
+/datum/map_template/surface/wilderness/normal/engi-station
```

Comment:
Travis might be angry about this? I've never seen a - in a type path.
Also an issue for the area name

Reply:
Ahh... I'll change that, thanks.